utils/create_command_docs.py

```diff
@@ -0,0 +1,191 @@
+#!/usr/bin/env python
+
+import importlib
+from pathlib import Path
+from typing import Any
+from typing import Generator
+from typing import List
+from typing import NamedTuple
+from typing import Optional
+
+import click
+
+TEMPLATE = """
+# {command_name}
+
+{parent}
+
+{description}
+
+## Usage
+
+```
+{usage}
+```
+
+## Options
+
+{options}
+
+{commands}
+
+## CLI Help
+
+```
+{help}
```

Comment:
I think the stuff above with the heading basically duplicates this help output, so we probably only need one. With the headings seems nice in the context of a web page.

Reply:
I was thinking the same actually. This bit is quite likely redundant - will remove this in a subsequent commit.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/83/commits/20bff77010b784638062155a21bd3886f4e24367.